### DISCUSSION TITLE
bobby mod support?

### DISCUSSION THREAD

**Discussion by Superdie02** [Sinytra/Connector#1400] [Q&A]
do you guys support bobby or a workaround to using it without sodium?
here is the mod link incase you need it 
https://modrinth.com/mod/bobby

**Comment by Superdie02** [comment]
wait wrong mod sorry i meant iris

**Reply by digi2303** [reply]
Which version are you using?

**Reply by Superdie02** [reply]
which, iris or sinytra

**Reply by digi2303** [reply]
Minecraft

**Comment by Superdie02** [comment]
1.20.1

**Comment by Superdie02** [comment]
I'm sending via email

**Reply by digi2303** [reply]
Use Oculus instead of Iris

**Comment by Superdie02** [comment]
I did but for some reason when I use complimentary unbound
The shades don't work

**Reply by digi2303** [reply]
There is a very unstable version of Iris for 1.20.1 Forge in their Discord server.You can try it but its buggy.
